### PR TITLE
Add Nova View & Copy Path Buttons

### LINKED ISSUE #118: Nova View & Copy Path Buttons
Let's also add the Path & "View" buttons on Markets (#41) and Locations (#42) just like we did on Pages in the Blog package (https://github.com/drewroberts/blog/issues/87).

### PR DESCRIPTION
**Pull request by chx2**
Closes #118

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "require": {
         "php": "^7.4|^8.0",
         "drewroberts/blog": "^4.5.2",
+        "sixlive/nova-text-copy-field": "^1.5",
```
Comment (chx2) [inline composer.json:20]: Will need to update this simliary to https://github.com/drewroberts/blog/pull/88#discussion_r615536359

Comment (drewroberts) [reply composer.json:20]: Thank you. Can you create an issue on the blog package for it?

File: src/Nova/Location.php [hunk 2]
```diff
@@ -45,6 +55,9 @@ public function fields(Request $request)
             nova('market') ? BelongsTo::make('Market', 'market', nova('market'))->required() : null,
             Text::make('Name')->required(),
             Slug::make('Slug')->from('Name'),
+            TextCopy::make('Link',  function () {
+                return config('app.url') . config('tipoff.web.uri_prefix') . $this->path;
```
Comment (drewroberts) [inline src/Nova/Location.php:59]: We should probably create another accessor that returns the full webpage url for the path.

**Review by drewroberts — APPROVED** [review]
Thank you! 🚠

**Pull request merged**